test/e2e/nodes/nodes.go

```diff
@@ -1 +1,48 @@
 package nodes
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/openshift-kni/mixed-cpu-node-plugin/test/e2e/pods"
+)
+
```

Comment:
this content is related to nodes but seems too loosely coupled to belong there. The code looks OK though.

You may want to extract the utility/library code out of the `test/e2e` hierarchy, so you can add unit tests when/if makes sense.

Reply:
hmm, so something like `utills/pods` `utils/nodes` etc. ?

Reply:
[the sooner we stop `utils` the better](https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common) :)

Reply:
I'd go for either `internal` or `pkg` trees